python/lsst/obs/ctio0m9/ctio0m9Mapper.py

```diff
@@ -93,10 +101,47 @@ def _computeCcdExposureId(self, dataId):
         visit = dataId['visit']
         return int(visit)
 
-    def std_raw(self, item, dataId):
-        item = super(Ctio0m9Mapper, self).std_raw(item, dataId)
+    def bypass_ccdExposureId(self, datasetType, pythonType, location, dataId):
+        """Hook to retrieve identifier for CCD"""
+        return self._computeCcdExposureId(dataId)
+
+    def bypass_ccdExposureId_bits(self, datasetType, pythonType, location, dataId):
+        """Hook to retrieve number of bits in identifier for CCD"""
+        return 32
+
+    def std_raw_md(self, md, dataId):
```

Comment:
As stated in the ticket, I think this should actually call the same free function that `translate_taiDate` uses so that we do not need the change to `pipe_tasks`.